labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -27,7 +36,14 @@ object ListManipulationExercise02 {
    * method above
    */
   def sumOfMany(l: List[Int]*): List[Int] = {
-    error("fix me")
+    val listOfLists = l.toList
+    def sumHelper(list:List[List[Int]]):List[Int] = {
+      list match{
+        case x::xs => sumOfTwo(x,sumHelper(xs))
```

Comment:
I was going to ask why you (and many others) named the rest of the list xs (first::rest was the convention I learned), but then I googled it and found the [most obvious answer](http://stackoverflow.com/questions/3938031/why-are-list-and-string-identifiers-named-xs-in-scala-and-other-languages). Now I feel silly.